xml/System.Diagnostics/ProcessStartInfo.xml

```diff
@@ -279,6 +279,7 @@
 ## Remarks  
  If the <xref:System.Diagnostics.ProcessStartInfo.UseShellExecute%2A> property is `true` or the <xref:System.Diagnostics.ProcessStartInfo.UserName%2A> and <xref:System.Diagnostics.ProcessStartInfo.Password%2A> properties are not `null`, the <xref:System.Diagnostics.ProcessStartInfo.CreateNoWindow%2A> property value is ignored and a new window is created.  
   
+ .NET Core does not support creating windows directly on Unix-like platforms, including macOS and Linux. This property is ignored on such platforms.
```

Comment:
This documentation page says that it also applies to Xamarin Android, iOS and Mac. Do you know if the behavior is the same on those? And if it is, should your note say that is applies to all "non-Windows" platforms?

Reply:
Sorry, I don't know :(